gcp_variant_transforms/testing/integration/large_tests/platinum_no_merge.json

```diff
@@ -0,0 +1,22 @@
+{
+  "test_name": "platinum-no-merge",
+  "table_name": "platinum_no_merge",
+  "input_pattern": "gs://genomics-public-data/platinum-genomes/vcf/*.vcf",
+  "runner": "DataflowRunner",
+  "worker_machine_type": "n1-standard-16",
+  "max_num_workers": "5",
+  "assertion_configs": [
+    {
+      "query": ["NUM_ROWS_QUERY"],
+      "expected_result": {"num_rows": 309551691}
+    },
+    {
+      "query": ["SUM_START_QUERY"],
+      "expected_result": {"sum_start": 24107891293483855}
```

Comment:
huh, I did not know BQ can actually handle such large sums! cool!

Reply:
Indeed :-)